tendrl/gluster_integration/sds_sync/event_utils.py

```diff
@@ -27,7 +21,7 @@ def emit_event(resource, curr_value, msg, instance):
         integration_id=NS.tendrl_context.integration_id,
         cluster_name=NS.tendrl_context.cluster_name,
         sds_name=NS.tendrl_context.sds_name,
-        fqdn=socket.getfqdn()
+        fqdn=NS.node_context.fqdn
     )
     alert['node_id'] = NS.node_context.node_id
     if not NS.node_context.node_id:
```

Comment:
For cluster related alert node_id is not mandatory

Reply:
We should make the node_id mandatory, Every alert even inside a cluster will come from some node_id